config/default/manager_auth_proxy_patch.yaml

```diff
@@ -23,3 +23,4 @@ spec:
         args:
         - "--metrics-addr=127.0.0.1:8080"
         - "--enable-leader-election"
+        - "--reconcile-timeout=2m"
```

Comment:
I think reconciliation-timeout would be clearer. reconcile-timeout gives the impression that timeouts are what is being reconciled.